src/components/CaseStudies/CaseStudiesHeader.tsx

```diff
@@ -0,0 +1,86 @@
+import { Trans, t } from '@lingui/macro'
+import { Button } from 'antd'
+import SectionHeader from 'components/SectionHeader'
+import Image from 'next/image'
+import Link from 'next/link'
+import { classNames } from 'utils/classNames'
+import { formattedNum, parseWad } from 'utils/format/formatNumber'
+import ETHAmount from '../currency/ETHAmount'
+import {
+  CaseStudiesConfiguration,
+  CaseStudiesConfigurationProps,
+} from './CaseStudiesConfiguration'
+
+interface CaseStudyHeaderProps {
+  header: string
+  title: string
+  subtitle: string | JSX.Element
+  totalRaised: number | string
+  totalPayments: number
+  summary: string | JSX.Element
+  projectUrl: string
+  fundingConfig: CaseStudiesConfigurationProps
+}
+
+export const CASE_STUDY_BODY_TEXT_COLOR = 'text-grey-700 dark:text-slate-200'
+
+export function CaseStudiesHeader({
+  header,
+  title,
+  subtitle,
+  totalRaised,
+  totalPayments,
+  summary,
+  projectUrl,
+  fundingConfig,
+}: CaseStudyHeaderProps) {
+  const minorHeadingClasses = `text-sm uppercase ${CASE_STUDY_BODY_TEXT_COLOR}`
+  const bigStatsContainer = 'flex flex-col items-start'
+  const bigStats = 'text-4xl font-display'
```

Comment:
@johnnyd-eth try to avoid this - bit of an anti-pattern.

Favor creating components for each piece of UI instead.

Reply:
Done, good call